source/puppet/5.3/hiera_migrate.md

```diff
@@ -69,70 +69,73 @@ You can enable the environment layer gradually. In migrated environments, the le
 
-   Save the resulting file as `<ENVIRONMENT>/hiera.yaml`. For example, `/etc/puppetlabs/code/environments/production/hiera.yaml`.
+8.  If any of your data relies on custom backends that have been ported to Hiera 5, install them in the environment. Hiera 5 backends are distributed as Puppet modules, so each environment can use its own version of them.
 
-8. If any of your data relies on custom backends that have been ported to Hiera 5, install them in the environment. Hiera 5 backends are distributed as Puppet modules, so each environment can use its own version of them.
+9.  If you use only file-based Hiera 5 backends, move the environment's data directory by renaming it from its old name (`hieradata`) to its new name (`data`). If you use custom file-based Hiera 3 backends, the global layer still needs access to their data, so you need to sort the files: Hiera 5 data moves to the new data directory, and Hiera 3 data stays in the old data directory. When you have Hiera 5 versions of your custom backends, you can move the remaining files to the new datadir.
 
-9. If you use only file-based Hiera 5 backends, move the environment’s data directory by renaming it from its old name (`hieradata`) to its new name (`data`). If you use custom file-based Hiera 3 backends, the global layer still needs access to their data, so you need to sort the files: Hiera 5 data moves to the new data directory, and Hiera 3 data stays in the old data directory. When you have Hiera 5 versions of your custom backends, you can move the remaining files to the new datadir.
+    If you use non-file backends that don't have a data directory:
 
-   If you use non-file backends that don’t have a data directory:
-   * Decide that the global hierarchy is the right place for configuring this data, and leave it there permanently.
-   * Do something equivalent to moving the datadir; for example, make a new database table for migrated data and move values  into place as you migrate environments.
-   * Allow the global and environment layers to use duplicated configuration for this data until the migration is done. 
+    -   Decide that the global hierarchy is the right place for configuring this data, and leave it there permanently.
+    -   Do something equivalent to moving the datadir; for example, make a new database table for migrated data and move values  into place as you migrate environments.
+    -   Allow the global and environment layers to use duplicated configuration for this data until the migration is done.
 
-10. Repeat these steps for each environment. If you manage your code by mapping environments to branches in a control repo, you can migrate most of your environments using your version control system’s merging tools. 
+10. Repeat these steps for each environment. If you manage your code by mapping environments to branches in a control repo, you can migrate most of your environments using your version control system's merging tools.
```

Comment:
Here's why the indents are four characters: that's the minimum indentation for an ordered list with 10 or more items.